kratos/input_output/vtk_output.cpp

```diff
@@ -124,12 +124,12 @@ void VtkOutput::PrintOutput()
 /***********************************************************************************/
 /***********************************************************************************/
 
-void VtkOutput::WriteModelPartToFile(const ModelPart& rModelPart, const bool IsSubModelPart)
+void VtkOutput::WriteModelPartToFile(const ModelPart& rModelPart, const bool IsSubModelPart, const std::string& rOutputFilename)
 {
     Initialize(rModelPart);
 
     // Make the file stream object
-    const std::string output_file_name = GetOutputFileName(rModelPart, IsSubModelPart);
+    const std::string output_file_name = (rOutputFilename == "") ? GetOutputFileName(rModelPart, IsSubModelPart) : rOutputFilename;
```

Comment:
Why completely replacing the file name , but why not using the input as a prefix to the name from GetOutputFileName ?

Reply:
the prefix is not flexible enough - imagine you want to change the it during runtime => not possible

maybe adding the folder would be an option, I don't have a strong opinion

Reply:
ok .. I was more like using the input string as prefix. 
But ya it may not be flexible. 
But using it as a prefix would lift some responsibility from the user (adding all the rank , step and modelpart name to the name). That was my motivation for proposing prefix.

Reply:
that does make sense indeed, I also was thinking abt wank etc
so what is passed to `PrintOutput` would be an additional label in the overall filename?

Reply:
> so what is passed to PrintOutput would be an additional label in the overall filename?

exactly